.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-beta.5"]
```

Comment:
I recommend "N.N-dev" for all versions, as [recommended by the action maintainers](https://github.com/actions/setup-python/issues/213).

Reply:
Sounds good, will do!